examples/data-sources/opslevel_domains/data-source.tf

```diff
@@ -1,6 +1,5 @@
-data "opslevel_domains" "all" {
-}
+data "opslevel_domains" "all" {}
 
 output "found" {
   value = data.opslevel_domains.all
```

Comment:
Isn't this syntax now incorrect?

Reply:
No, this is the same. A quick `terraform plan` can prove it works.

Reply:
@davidbloss you're right, this one is fine but there are some other ones that are invalid